src/index.html

```diff
@@ -348,8 +346,67 @@ <h2 class="social-media-links__title">Let&apos;s connect</h2>
       </section>
 
       <!-- Contact Form -->
-      <section class="container">
-        <!-- Social Media Links -->
+      <section class="container contact-me">
+        <h2 class="contact-me__title">Contact me</h2>
+
+        <p class="contact-me__description">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it!
+        </p>
+
+        <form
+          class="contact-me__form"
+          action="https://formspree.io/f/mqkonbkk"
+          method="POST"
+        >
+          <div>
+            <label class="contact-me__form__label" for="full-name">
+              Full name:</label
+            >
+            <input
+              class="contact-me__form__input"
+              type="text"
+              id="full-name"
+              name="fullName"
+              placeholder="Alexandra Loremipsum"
+              maxlength="30"
+              required
+            />
+          </div>
+          <div>
+            <label class="contact-me__form__label" for="email">Email: </label>
+            <input
+              class="contact-me__form__input"
+              type="email"
+              id="email"
+              name="email"
+              placeholder="alexandra@gmail.com"
+              required
+            />
+          </div>
+          <div>
+            <label class="contact-me__form__label" for="message">
+              Message:
+            </label>
+            <textarea
+              class="contact-me__form__input contact-me__form__textarea"
+              name="message"
+              id="message"
+              placeholder="Write your message here"
+              maxlength="500"
+            ></textarea>
```

Comment:
- Kindly consider adding the `required` attribute on the `textarea` input field as this will make sure that a user will be unable to submit the form if any of the input fields are empty as specified in the project requirement. 
<img width="664" alt="Screenshot 2023-05-02 at 20 14 33" src="https://user-images.githubusercontent.com/107192076/235776129-dc12012b-f475-4485-ac39-6e715a365c12.png">


- This attribute helps to validate the user's `input` and alerts them if they leave the `input` field empty. Adding the `required` attribute will ensure that the user fills in the `input` field before submitting the `form`, which can improve the usability and user experience of your website. See below for more details: